parsers/ENTSOE.py

```diff
@@ -620,18 +620,22 @@ def parse_production(xml_text):
         is_production = len(timeseries.find_all('inBiddingZone_Domain.mRID'.lower())) > 0
         psr_type = timeseries.find_all('mktpsrtype')[0].find_all('psrtype')[0].contents[0]
 
-        if len(timeseries.find_all('inBiddingZone_Domain.mRID'.lower())) > 0:
-            for entry in timeseries.find_all('point'):
-                quantity = float(entry.find_all('quantity')[0].contents[0])
-                position = int(entry.find_all('position')[0].contents[0])
-                datetime = datetime_from_position(datetime_start, position, resolution)
-                try:
-                    i = datetimes.index(datetime)
+        for entry in timeseries.find_all('point'):
+            quantity = float(entry.find_all('quantity')[0].contents[0])
+            position = int(entry.find_all('position')[0].contents[0])
+            datetime = datetime_from_position(datetime_start, position, resolution)
+            try:
+                i = datetimes.index(datetime)
+                if is_production:
                     productions[i][psr_type] += quantity
-                except ValueError:  # Not in list
-                    datetimes.append(datetime)
-                    productions.append(defaultdict(lambda: 0))
-                    productions[-1][psr_type] = quantity if is_production else -1 * quantity
+                elif psr_type in ENTSOE_PARAMETER_GROUPS['storage']['hydro storage']:
```

Comment:
Shouldn't we ignore all types of storage? Otherwise, if another storage type is added to ENTSO-E (e.g. battery storage) we'll run into the same problem again

Reply:
Hmm, I guess it makes sense that it would also apply for other types of storage from reading through some of the other issues. What do you think @willbeaufoy?

Reply:
Good idea, I've made a new pull request to do that here:

https://github.com/tmrowco/electricitymap-contrib/pull/2812